requirements.txt

```diff
@@ -1,7 +1,7 @@
 -e git+https://github.com/ckan/ckanext-report.git@master#egg=ckanext-report
 -e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest
 
-ckanext-datagovtheme
+-e git+https://github.com/gsa/ckanext-datagovtheme.git@ckan-210#egg=ckanext-datagovtheme
```

Comment:
We should probably merge https://github.com/GSA/ckanext-datagovtheme/pull/162/files, and then leave this line as it was...

Reply:
+1